index.js

```diff
@@ -76,9 +76,13 @@ program
 
 if ((os.platform() === 'win32' && require('is-administrator')()) || process.getuid() <= 0) {
     async.waterfall([(cb) => {
-        return properties.parse(config.CONF_FILE, {
-            path: true
-        }, cb);
+        if (fs.existsSync(config.CONF_FILE)) {
```

Comment:
I'd suggest to do it in the following way:
```
        return properties.parse(config.CONF_FILE, {
            path: true
        }, (error, parsedConfig) => {
            return cb(null, parsedConfig);
        })
```

which is `callback` version of [this](https://github.com/logdna/logdna-agent/blob/baf34affe3fc74bdcd16fc2e1bf0ee68daf55abe/index.js#L98-L99).